sdk/storage/Azure.ResourceManager.Storage/api/Azure.ResourceManager.Storage.netstandard2.0.cs

```diff
@@ -1412,6 +1433,25 @@ public partial class LegalHoldTag
         public System.DateTimeOffset? Timestamp { get { throw null; } }
         public string Upn { get { throw null; } }
     }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct ListEncryptionScopesInclude : System.IEquatable<Azure.ResourceManager.Storage.Models.ListEncryptionScopesInclude>
```

Comment:
Could we rename this to `EncryptionScopesIncludeType`?
Only suggestion, please feel free to change it to names you feel reasonable

Reply:
Done